### PR TITLE
Issue 1215: A script to automatically rebuild virtualenv_support 

### PR DESCRIPTION
**Pull request by ThomasTPG**

### CODE REVIEW

File: bin/rebuild-support.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```
Comment (gaborbernat) [inline bin/rebuild-support.py]: we'll not call this with the default python, but instead be explicit and pass the script to the proper Python as such we don't need shebang lines.

File: bin/rebuild-support.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+"""
+Helper script to rebuild virtualenv_support. Downloads the wheel files using pip
+"""
+
+import glob
+import os
+import subprocess
+
+def removeWheelFiles():
+    for file in glob.glob("../src/virtualenv_support/*.whl"):
```
[the remark this answers is not in the document]
Comment (gaborbernat) [reply bin/rebuild-support.py]: Oh and please use snake case names as outlined in PEP-8.

**Comment by ThomasTPG** [comment]
#1215

**Comment by gaborbernat** [comment]
@ThomasTPG please allow changes from maintainers https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by gaborbernat** [comment]
thanks @ThomasTPG

**Pull request merged**